keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c

```diff
@@ -15,16 +15,16 @@
  * along with this program. If not, see <https://www.gnu.org/licenses/>.
```

Comment:
Why move this from the community folder, that seems like taking a step backwards.

Reply:
Based on @noroadsleft comment on [#10338](https://github.com/qmk/qmk_firmware/pull/10338#discussion_r492310612), matrix wiring for the board being used with the keymap are accurate for the Soyuz, but some boards with this layout wire things differently. I'm afraid in the future if i have another numpad 5x4 pcb again, it won't be work. CMIIW @zvecr.

Reply:
Matrix wiring doesn't matter, that's the whole point of abstraction within layout macros. Anything that supports `numpad_5x4` will work with the keymap.

Reply:
Oh ok, then i will move again to the community folder.